pylint/checkers/refactoring/recommendation_checker.py

```diff
@@ -337,7 +337,7 @@ def visit_const(self, node: nodes.Const) -> None:
 
     def _detect_replacable_format_call(self, node: nodes.Const) -> None:
         """Check whether a string is used in a call to format() or '%' and whether it
-        can be replaced by a f-string"""
+        can be replaced by an f-string"""
```

Comment:
I'm doubting myself here, but should this be `an`?

Reply:
Yes, I would say "an eff string"

Reply:
Thougth the same as @jacobtylerwalls, it's a tricky one :smile:

Reply:
Ah yeah! My bad, thanks @jacobtylerwalls!